examples/music_drop_fade.py

```diff
@@ -168,9 +168,7 @@ def main():
     clock = pg.time.Clock()
 
     pg.scrap.init()
-    pg.SCRAP_TEXT = pg.scrap.get_types()[0]  # TODO remove when scrap module is fixed
-    clipped = pg.scrap.get(pg.SCRAP_TEXT).decode("UTF-8")
-    # store the current text from the clipboard TODO remove decode
+    clipped = pg.scrap.get(pg.SCRAP_TEXT)
```

Comment:
scrap is still not "fixed" yet... this is a problematic module in pygame
Which system are you running this example on? I just tested this example on windows and ubuntu and it works fine (infact this change would probably break the example on these platforms)

Reply:
I tested it on Windows-10-10.0.19044-SP0
And pygame 2.1.2

Reply:
Oh, I can reproduce the example being broken on 2.1.2 windows release... but when I said it works on windows, I was using the latest `pygame:main` branch and it does work here for some reason (my best guess is that it has to do with the SDL version change, AFAIK no code changes happened to scrap in the last few months)

Reply:
This got to be working on all platforms, I tested it on Debian too, it works.

Reply:
I mean shorted code.

Reply:
`pg.scrap.get` always returns a `bytes` instance... so it has to be decoded to `str` because functions processing it expect it to be a string
I get this traceback when I run your code 
```
Traceback (most recent call last):
  File "D:\gitstuff\pygame\examples\music_drop_fade.py", line 246, in <module>
    main()
  File "D:\gitstuff\pygame\examples\music_drop_fade.py", line 237, in main
    play_file(clipped)  # try to play the file if it has
  File "D:\gitstuff\pygame\examples\music_drop_fade.py", line 59, in play_file
    if add_file(filename):
  File "D:\gitstuff\pygame\examples\music_drop_fade.py", line 33, in add_file
    if filename.rpartition(".")[2].lower() not in music_file_types:
TypeError: a bytes-like object is required, not 'str'
```
I get this when I copy a new string onto my clipboard